src/jpegcodec.c

```diff
@@ -344,19 +352,8 @@ gdip_load_jpeg_image_internal (struct jpeg_source_mgr *src, GpImage **image)
 		if (cinfo.num_components == 1)
 			result->active_bitmap->palette = gdip_create_greyscale_palette (256);
 		break;
-	case JCS_RGB:
-		result->active_bitmap->image_flags |= ImageFlagsColorSpaceRGB;
-		break;
-	case JCS_YCbCr:
-		result->active_bitmap->image_flags |= ImageFlagsColorSpaceYCBCR;
-		break;
-	case JCS_YCCK:
-		result->active_bitmap->image_flags |= ImageFlagsColorSpaceYCCK;
-		break;
-	case JCS_CMYK:
-		result->active_bitmap->image_flags |= ImageFlagsColorSpaceCMYK;
-		break;
```

Comment:
Removing this doesn't sound right. The failed test is weird though:

> Bitmap24bitFeatures (MonoTests.System.Drawing.Imaging.JpegCodecTest.Bitmap24bitFeatures)
>      Flags
>   Expected: 77960
>   But was:  77840

The expected value differs by combination of ImageFlagsColorSpaceYCCK and ImageFlagsColorSpaceCMYK.

Reply:
Eh, sorry, my bad... it's not hex :)

Reply:
Expected: ImageFlagsReadOnly | ImageFlagsHasRealPixelSize | ImageFlagsHasRealDPI | ImageFlagsColorSpaceYCBCR | ImageFlagsPartiallyScalable
But was: ImageFlagsReadOnly | ImageFlagsHasRealPixelSize | ImageFlagsHasRealDPI | ImageFlagsColorSpaceRGB

Reply:
Also expected according to GDI+